03/app.js

```diff
@@ -0,0 +1,15 @@
+const buttons = document.querySelectorAll('button');
+
+if (!buttons || !buttons.length) {
```

Comment:
Tutaj błąd! `!buttons` - nic nie daje. `.querySelectorAll` - zawsze zwraca tablicę, tablica to obiekt, a obiekt zawsze jest prawdziwy (nawet pusty/bez zawartości). Dlatego wystarczy `!buttons.length` - nie wiem czy nie czytelniej będzie `buttons.length === 0`

Reply:
To jest czepialstwo! Ale jasne, zrozumiałe :)